### ISSUE TITLE
CTHUB - vinpower connection

### ISSUE THREAD

**Issue by tim738745** [bcgov/cthub#273]
**Describe the task**
Get the connection/configuration information/credentials for vinpower, so that, in the future (another card), we can Integrate vinpower with the vin decoder (https://github.com/bcgov/vdcd)

**Purpose**
This is the first step towards allowing us to decode vins using vinpower.

**Acceptance Criteria**
- [ ] Having, in our possession, the information needed to connect to/call vinpower.

**Comment by kcabhar** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Team-Zelda-637d5bc12e6201001055b745/issues/bcgov/cthub/273?planning-poker) with Zenhub @emi-hi @JulianForeman @tim738745

**Comment by kcabhar** [comment]
Waiting on credentials - Jenna will be in touch after she follows up
